resources/themes/cura-light/theme.json

```
@@ -83,10 +83,22 @@
 
 
         "primary": [50, 130, 255, 255],
+        "primary_shadow": [64, 47, 205, 255],
         "primary_hover": [48, 182, 231, 255],
         "primary_text": [255, 255, 255, 255],
         "border": [127, 127, 127, 255],
         "secondary": [245, 245, 245, 255],
+        "secondary_shadow": [228, 228, 228, 255],
+
+        "primary_button": [38,113,231,255],
```

Code-style: Spaces after comma `,`
```suggestion
        "primary_button": [38, 113, 231, 255],
```